yessine.py

```diff
@@ -0,0 +1 @@
+print("hello world")
```

Comment:
Please use proper capitalisation and punctuation. 

```suggestion
print("Hello world!")
```

Reply:
it should be lower case